dev/io.openliberty.mcp.internal_fat/fat/src/io/openliberty/mcp/internal/fat/utils/HttpTestUtils.java

```diff
@@ -17,7 +17,27 @@
  */
 public class HttpTestUtils {
 
+    /**
+     * Call MCP server, and get the String response body
+     */
     public static String callMCP(LibertyServer server, String path, String jsonRequestBody) throws Exception {
         return new HttpRequest(server, path + "/mcp").requestProp("Accept", "application/json, text/event-stream").jsonBody(jsonRequestBody).method("POST").run(String.class);
-    };
-};
+    }
+
+    /**
+     * Call MCP server notification endpoint, and provide a 202 expected response code. No response body is returned
+     * If a response body is returned, or a response code that is not 202, and exception is thrown
+     */
+    public static void callMCPNotification(LibertyServer server,
+                                           String path,
+                                           String jsonRequestBody) throws Exception {
+
+        String response = new HttpRequest(server, path + "/mcp")
+                .requestProp("Accept", "application/json, text/event-stream")
+                .jsonBody(jsonRequestBody)
+                .method("POST")
+                .expectCode(202)
+                .run(String.class);
+        assert response == null;
```

Comment:
In general we use JUnit's [Assert](https://junit.org/junit4/javadoc/latest/org/junit/Assert.html) methods or the hamcrest [assertThat](https://hamcrest.org/JavaHamcrest/javadoc/1.3/org/hamcrest/MatcherAssert.html#assertThat(java.lang.String,%20T,%20org.hamcrest.Matcher)) so that a message is provided with the failure.

```suggestion
        assertNull("Notification request received a response", response);
```

Reply:
Fixed, Thanks